apps/meteor/client/views/blocks/MessageBlock.js

```diff
@@ -13,8 +17,18 @@ const mrkdwn = ({ text } = {}) => text && <span dangerouslySetInnerHTML={{ __htm
 messageParser.mrkdwn = mrkdwn;
 function MessageBlock({ mid: _mid, rid, blocks, appId }) {
 	const { ref, className } = useBlockRendered();
+	const setModal = useSetModal();
+	const joinCall = useJoinCall();
+	const room = useUserRoom(rid);
+
+	const handleCloseModal = useMutableCallback(() => setModal());
+
 	const context = {
-		action: ({ actionId, value, blockId, mid = _mid }) => {
+		action: ({ actionId, value, blockId, mid = _mid, appId }) => {
+			if (appId === 'videoconf-core' && actionId === 'join') {
```

Comment:
I changed this to check the parent appId since the join button is no longer the first block in the message